Tests.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+
+public class Tests
+{
+    static void ExpectTrue(bool expression)
+    {
+        if (!expression)
+        {
+            Console.WriteLine("Expected true, but got false");
+            Environment.Exit(1);

```

Comment:
Exit calls for termination of program, which in turn masks other tests that are next in line for execution.
Since this are only tests, it is convenient to know how many tests have failed or passed rather than stopping at the first instance of failure.